src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -4009,9 +4006,14 @@ namespace EnergyPlus::ZoneEquipmentManager {
                 Node(ZoneNode).MassFlowRateMin = TotInletAirMassFlowRateMin;
                 Node(ZoneNode).MassFlowRateMinAvail = TotInletAirMassFlowRateMinAvail;
 
-                // Calculate standard return air flow rate using default method of inlets minus exhausts adjusted for "balanced" exhaust flow
-                StdTotalReturnMassFlow = TotInletAirMassFlowRate + ZoneMixingNetAirMassFlowRate -
-                                         (TotExhaustAirMassFlowRate - state.dataZoneEquip->ZoneEquipConfig(ZoneNum).ZoneExhBalanced);
+                if (ZoneAirMassFlow.EnforceZoneMassBalance) {
+                    // do not include adjusted for "balanced" exhaust flow
+                    StdTotalReturnMassFlow = TotInletAirMassFlowRate + ZoneMixingNetAirMassFlowRate - TotExhaustAirMassFlowRate;
+                } else {
+                    // Calculate standard return air flow rate using default method of inlets minus exhausts adjusted for "balanced" exhaust flow
+                    StdTotalReturnMassFlow = TotInletAirMassFlowRate + ZoneMixingNetAirMassFlowRate -
+                        (TotExhaustAirMassFlowRate - state.dataZoneEquip->ZoneEquipConfig( ZoneNum ).ZoneExhBalanced);
+                }
                 if (!ZoneAirMassFlow.EnforceZoneMassBalance) {
```

Comment:
Do not include the "balanced" exhaust flow term in the zone air mass balancing enforcing equation if "EnforceZoneMassBalance" is true.

Reply:
Probably I should use "ZoneAirMassFlow.ZoneFlowAdjustment != DataHeatBalance::NoAdjustReturnAndMixing" to allow the existing equation when using infiltration treatment methods only to enforce zone air flow balance.

Reply:
This will be revised in the next push cycle along with unit tests.